components/tikv_util/src/worker/future.rs

```diff
@@ -104,11 +105,12 @@ where
     let metrics_handled_task_count = WORKER_HANDLED_TASK_VEC.with_label_values(&[name]);
 
     let handle = LocalSet::new();
+    let mut rt = Runtime::new().unwrap();
```

Comment:
`Runtime::new()` will create a default threaded runtime with some threads. The default value is the number of cores available to the system. FYI https://docs.rs/tokio/0.2.22/src/tokio/runtime/mod.rs.html#341-352

Reply:
I do not understand. Do you mean that we can only use `Basic Scheduler`?